content/YubiHSM2/Releases/Known_issues.adoc

```diff
@@ -8,6 +8,8 @@ VM USB redirection may function inconsistently, particularly in Windows environm
 
 === YubiHSM Shell
 
+ - The yubihsm-shell utility is limited to reading 4096 bytes in a single command.
```

Comment:
Please remove the - here, the other headings don't have lists when there's just one thing.